runtime/vm/criuhelpers.cpp

```diff
@@ -61,16 +61,17 @@ jvmCheckpointHooks(J9VMThread *currentThread)
 BOOLEAN
 jvmRestoreHooks(J9VMThread *currentThread)
 {
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
Can we put this entire function under a single `#ifdef`? There is no reason for this function here to be called if the CRIU flag is not enabled.

Reply:
Sure, I can do that (I was just maintaining the set of functions defined).

Reply:
The file is now only compiled if CRIU support is configured.